src/consumed.py

```diff
@@ -314,6 +319,12 @@ def output_results(self):
                     time_dat.datetime_local.dt.year == self.year
                 ]  # keep year of local data
 
+                # if there are small gaps in the consumed data,
+                # linearly interpolate the missing values, up to a maximum of 2 consecutive missing hours
+                time_dat.loc[:, CONSUMED_EMISSION_RATE_COLS] = time_dat.loc[
+                    :, CONSUMED_EMISSION_RATE_COLS
+                ].interpolate(method="linear", axis=0, limit=2, limit_area="inside")
```

Comment:
In the hourly consumed calculations, there are certain hours with missing data. I'm not sure if this has always been happening and I've just never noticed, or whether this is a new issue that just started happening. It looks like in the 2020 data posted online, these gaps are less frequent or not occuring at all. I proposed this linear interpolation as a patch, but not sure if we should try and track down the root cause of these missing values? @gailin-p  what do you think?

Reply:
What's the frequency of the missing hours? If they're much more frequent in the 2021 data then I think we should try to figure out why. 
Also, are the same # of hours missing in the posted 2020 data as in 2020 data computed using the current code? If there are more using the current code, that also seems like a warning sign

Reply:
Let me look into this but I think the frequency is greater.

Reply:
Ok the existing consumed emissions data for 2020 posted online has 168 missing values and looks like this when visualized for missing values: 
![image](https://user-images.githubusercontent.com/45949268/209711492-78935a3e-345e-4e6a-895d-ac5f1d8dd22c.png)
The new consumed emissions data has 5562 missing values, most of which appear to be coming from EEI and SEC (which are coincidentally the only two BAs now listed in `BA_930_INCONSISTENCY`):
![image](https://user-images.githubusercontent.com/45949268/209711600-c4a47609-42bb-4477-beaa-3795b1d4e135.png)

Reply:
Testing this for the 2021 data, I'm now discovering that the number of missing values is not stable, and it changes each time the consumed emissions calc is run. Previously, I was seeing a handful of missing values in each file, and in the couple of times I've run this locally this morning, sometimes I am getting no missing values, sometimes I am getting one hour with missing values, sometimes I am getting two hours. This seems like an issue that this isn't stable... does this maybe have to do with how we are perturbing the calculation?

Reply:
It turns out that this issue was caused by using the MKL build of the `blas` package, as documented in https://github.com/numpy/numpy/issues/20233 and https://github.com/conda/conda/issues/11023. Essentially what was happening was that when we were inverting the matrix, we were getting inconsistent results. This was fixed by changing the environment to include `nomkl` so that the `openblas` version of `blas` is installed.